packages/bundler-plugin-core/src/index.ts

```diff
@@ -116,13 +119,17 @@ const unplugin = createUnplugin<Options>((options, unpluginMetaContext) => {
     /**
      * Responsible for starting the plugin execution transaction and the release injection span
      */
-    buildStart() {
+    async buildStart() {
+      if (!internalOptions.release) {
+        internalOptions.release = await cli.releases.proposeVersion();
+      }
```

Comment:
In case we don't have a release (i.e. an empty sting, we use the CLI to detect it automatically)